.github/workflows/ci.yml

```diff
@@ -22,3 +21,43 @@ jobs:
           skip_install: 'true'
           use_bundler: 'true'
       - run: test "$(bundle exec rubocop --version)" == "1.18.1"
+  test-only_changed:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    env:
+      INPUT_ONLY_CHANGED: 'true'
+    steps:
+      - uses: actions/checkout@v4
+      - name: setup
+        run: |
+          git config user.email "workflow@github.com"
+          git config user.name "I am an automated workflow"
+      - name: Check when there are relevant files
+        run: |
+          git checkout ${{ github.sha }}
+          cp test/only_changed/few_relevant/files/* .
+          git add *
+          git commit -m auto
+
+          export PATH=test/only_changed/few_relevant/mock_bins:test/only_changed/shared_mock_bins:$PATH
+          BASE_REF=$(git rev-parse HEAD~) HEAD_REF=$(git rev-parse HEAD) ./script.sh
+      - name: Check when there are no relevant files
+        run: |
+          git checkout ${{ github.sha }}
+          cp test/only_changed/nothing_relevant/files/* .
+          git add *
+          git commit -m auto
+
+          export PATH=test/only_changed/nothing_relevant/mock_bins:test/only_changed/shared_mock_bins:$PATH
+          BASE_REF=$(git rev-parse HEAD~) HEAD_REF=$(git rev-parse HEAD) ./script.sh
+      - name: Check when there are too many relevant files
+        run: |
+          git checkout ${{ github.sha }}
+          touch a{00..100}.rb
```

Comment:
Should we create 101 (more than 100) files here?

Reply:
The range is inclusive, so 101 files are generated, I can change to `{01..101}` if it will be clearer

Reply:
I accidentally tested that with exactly 100 it passes file names to rubocop: https://github.com/reviewdog/action-rubocop/actions/runs/9913233268/job/27389778004

Reply:
ok. 

I'm a bit confused, but why does the test result shows " No relevant files for rubocop, skipping"?
![image](https://github.com/user-attachments/assets/839d8ffe-903c-4cb9-b3f0-d5f24492f44b)
https://github.com/reviewdog/action-rubocop/actions/runs/9913690348/job/27391225472

Reply:
Great that you spotted it, I didn't notice that change from ruby to bash script for rubocop mock was producing files prefixed with `./`. I hardened the test by explicitly validating that reviewdog mock was or was not called.